### ISSUE TITLE
warning: 'getwd' is deprecated on Ubuntu

### ISSUE THREAD

**Issue by stanislaw** [mull-project/mull#178]
```
[29/48] Building CXX object lib/CMakeFiles/mull.dir/SQLiteReporter.cpp.o
../lib/SQLiteReporter.cpp:73:3: warning: 'getwd' is deprecated [-Wdeprecated-declarations]
  getwd(wd);
  ^
/usr/include/unistd.h:528:14: note: 'getwd' has been explicitly marked deprecated here
extern char *getwd (char *__buf)
             ^
1 warning generated.
```

**Comment by AlexDenisov** [comment]
Same on CentOS.

**Comment by AlexDenisov** [comment]
Fixed by #370
